docs/deprecations.md

```diff
@@ -6,6 +6,16 @@ layout: Doc
 
 # Serverless Framework Deprecations
 
+<a name="AWS_API_GATEWAY_SPECIFIC_KEYS"><div>&nbsp;</div></a>
+
+## `awsKmsKeyArn` references
```

Comment:
I believe this is a leftover, maybe it could say "API Gateway-specific configuration" or something along those lines?